ci/pipelines/smoke-tests.yml

```diff
@@ -0,0 +1,34 @@
+---
+groups:
+  - name: Smoke Tests
+    jobs:
+      - card-smoke-test
+
+resources:
+  - name: every-10m
+    type: time
+    source: { interval: 10m }
+
+  - name: omnibus
+    type: git
+    source:
+      uri: https://github.com/alphagov/pay-omnibus
+      branch: smoke_tests
+
+jobs:
+  - name: card-smoke-test
+    serial: true
+    plan:
+      - get: every-10m
+        trigger: true
+      - get: omnibus
+      - task: test
+        file: omnibus/ci/tasks/run-smoke-test.yml
+        params:
+          SMOKE_TEST: smoke-card
```

Comment:
Do we need to set the `PUBLICAPI_URL` and `SELFSERVICE_URL` vars here too? I'm looking at the params for `ci/tasks/run-smoke-test.yml` that seem to require setting:
```
params:
  SMOKE_TEST:
  PUBLICAPI_URL:
  SELFSERVICE_URL:
  SELENIUM_HUB_URL:
```

Reply:
I'm not 100% sure what params each smoke test needs just yet. These are just placeholders for now.

Reply:
oh right sorry, makes sense 👍